pkg/codegen/testing/test/README.md

```diff
@@ -0,0 +1,65 @@
+# SDK Codegen Tests
+
+TestSDKCodegen runs the complete set of SDK code generation tests
+against a particular language's code generator. It also verifies
+that the generated code is structurally sound.
+
+The test files live in `pkg/codegen/testing/test/testdata` and
+are registered in `var sdkTests` in `sdk_driver.go`.
```

Comment:
This bit is outdated now.
They're now registered in:

```
pkg/codegen/testing/test/sdk_driver.go
84:var PulumiPulumiSDKTests = []*SDKTest{

pkg/codegen/testing/test/program_driver.go
56:var PulumiPulumiProgramTests = []ProgramTest{
236:var PulumiPulumiYAMLProgramTests = []ProgramTest{
```

(These are exported because at least pulumi-java's codegen tests import and use this var directly.)

So maybe:
```suggestion
are registered in the following globals in pkg/codegen/testing/test.

- sdk_driver.go: `PulumiPulumiSDKTests`
- program_driver.go: `PulumiPulumiProgramTests`
- program_driver.go: `PulumiPulumiYAMLProgramTests`
```